cirq/ionq/calibration_test.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 The Cirq Developers
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+
+from cirq import ionq
+
+
+def test_calibration_fields():
+    calibration_dict = {
+        'connectivity': [
+            [0, 1],
+            [0, 2],
+            [1, 2],
+        ],
+        'target': 'ionq.qpu',
+        'qubits': 3,
+        'fidelity': {'1q': {'mean': 0.999}, '2q': {'mean': 0.999}},
+        'timing': {
+            '1q': 1.1e-05,
+            '2q': 0.00021,
+            'readout': 0.000175,
+            'reset': 3.5e-05,
+        },
+        'date': '2020-08-07T12:47:22.337Z',
+    }
+    cal = ionq.Calibration(calibration_dict=calibration_dict)
+    assert cal.connectivity() == ((0, 1), (0, 2), (1, 2))
```

Comment:
>  Note that this only returns one of (q0, q1) and (q1, q0) if q0 and q1 can interact.

Consider adding a test for this since it was asserted in the documentation.

Reply:
I think this is a contract of the api's returned values, so testing it just means mocking this property and then asserting it.  I don't think this really tests anything, does it?